dc/core/ContentWindow.cpp

```diff
@@ -110,6 +110,16 @@ void ContentWindow::setContent( ContentPtr content )
     createInteractionDelegate();
 }
 
+ContentWindowController* ContentWindow::getController()
```

Comment:
Actually I think it should remain a pointer, because a window has no controller until it has been added to a DisplayGroup.